Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php

```diff
@@ -77,14 +77,14 @@ public function testExtractForCreateItem()
                 AND `document_id` = :documentId
                 AND `status` = :status',
             [
-                'operationType' => 'c',
+                'operationType' => 'C',
```

Comment:
maybe it's better to use constants here, next time will be less find and replace..